### PR TITLE
dev-python/python-email-validator: new package of 1.1.1 version

### PR DESCRIPTION
**Pull request by xp-vit**
Adds new package dev-python/python-email-validator-1.1.1

Signed-off-by: Viktar Patotski <xp.vit.blr@gmail.com>

### CODE REVIEW

File: dev-python/email-validator/email-validator-1.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (mgorny) [inline dev-python/email-validator/email-validator-1.1.0.ebuild:6]: Please be more concise and group related vars together, i.e. DISTUTILS and PYTHON.

File: dev-python/email-validator/email-validator-1.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+MYPN="python-${PN}"
```
Comment (mgorny) [inline dev-python/email-validator/email-validator-1.1.0.ebuild:12]: Why are you renaming it in the first place? This is just confusing to everyone, and will become trouble once someone independently creates `email-validator` package (the name is free, isn't it?).

File: dev-python/email-validator/files/remove-tests-using-ns.patch [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+index 0ed1b36..deb9154 100644
+--- a/tests/test_main.py
++++ b/tests/test_main.py
+@@ -248,23 +248,3 @@ def test_email_invalid(email_input, error_msg):
+         validate_email(email_input)
+     # print(f'({email_input!r}, {str(exc_info.value)!r}),')
+     assert str(exc_info.value) == error_msg
+-
+-
+-def test_deliverability_no_records():
```
Comment (mgorny) [inline dev-python/email-validator/files/remove-tests-using-ns.patch:11]: We generally prefer `sed -i -e 's:test_foo:_&:'` kind of thing to disable tests. Causes less maintenance effort in the future.

Comment (mgorny) [reply dev-python/email-validator/files/remove-tests-using-ns.patch:11]: Also please file an upstream bug explaining why relying on internet is bad, and that this should be easily avoided by mocking.

File: dev-python/email-validator/metadata.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+
+<pkgmetadata>
+	<maintainer type="person">
+		<email>andrewammerlaan@riseup.net</email>
```
Comment (mgorny) [inline dev-python/email-validator/metadata.xml:6]: Did you ask him?

Comment (xp-vit) [reply dev-python/email-validator/metadata.xml:6]: Sure. And we agree to maintain it together.

File: dev-python/email-validator/files/remove-tests-using-ns.patch (hunk 3 above)
Comment (xp-vit) [reply dev-python/email-validator/files/remove-tests-using-ns.patch:11]: https://github.com/JoshData/python-email-validator/issues/42

File: dev-python/python-email-validator/metadata.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">JoshData/python-email-validator</remote-id>
+		<remote-id type="pypi">email-validator</remote-id>
+	</upstream>
+</pkgmetadata>
+
```
Comment (mgorny) [inline dev-python/python-email-validator/metadata.xml:22]: Oh no, trailing empty line!

File: dev-python/python-email-validator/python-email-validator-1.1.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+MYPN="${PN}"
```
Comment (mgorny) [inline dev-python/python-email-validator/python-email-validator-1.1.1.ebuild:11]: But… why?

File: dev-python/python-email-validator/python-email-validator-1.1.1.ebuild [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+LICENSE="CC0-1.0"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+
+RDEPEND="
+	>=dev-python/idna-2.0.0[${PYTHON_USEDEP}]
+	>=dev-python/dnspython-1.15.0[${PYTHON_USEDEP}]
+"
+
+#Patch removes tests which depend on network usage (availability if NS in particular).
```
Comment (mgorny) [inline dev-python/python-email-validator/python-email-validator-1.1.1.ebuild:28]: Remove it.

**Pull request closed without merging**